vnext/Shared/Networking/HttpFilterFactorySettings.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I do not think this functionality requires separate compilation unit.

Please move declarations and definitions to `vnext\Shared\Networking\WinRTHttpResource.{h,cpp}`.